src/overlays/actors/ovl_Bg_Treemouth/z_bg_treemouth.c

```diff
@@ -26,7 +26,10 @@ void func_808BC8B8(BgTreemouth* this, GlobalContext* globalCtx);
 void func_808BC9EC(BgTreemouth* this, GlobalContext* globalCtx);
 void func_808BCAF0(BgTreemouth* this, GlobalContext* globalCtx);
 
-#include "z_bg_treemouth_cutscene_data.c" EARLY
+extern CutsceneData D_808BCE20[];
+extern CutsceneData D_808BD2A0[];
+extern CutsceneData D_808BD520[];
+extern CutsceneData D_808BD790[];
```

Comment:
I think it's out of the scope of this PR but would we want to have these symbols in a `z_bg_treemouth_cutscene_data.h` header file

Thinking ahead about extraction (?) and tooling it seems to make sense

(also github ate this comment in my approval review)

Reply:
I agree this is a bit out of scope so we should probably discuss it in discord